vm/src/obj/objbytes.rs

```diff
@@ -201,3 +203,58 @@ fn bytes_iter(obj: PyBytesRef, _vm: &VirtualMachine) -> PyIteratorValue {
         iterated_obj: obj.into_object(),
     }
 }
+
+fn compare_slice(a: &[u8], b: &[u8]) -> bool {
```

Comment:
There should be a rust builtin function to compare slices: https://doc.rust-lang.org/nightly/std/slice/#trait-implementations in this case, this would mean this function is not required.